index.html

```diff
@@ -1,214 +1,212 @@
-
-
-      </section>
-      <!-- footer>
-        
-          <p>Project maintained by <a href="http://github.com/cxw42">cxw42</a></p>
-        
-        <p><small>Hosted on GitHub Pages &mdash; Theme modified from
-      <a href="https://github.com/pages-themes/leap-day">Leap Day</a> by
-      <a href="https://twitter.com/michigangraham">mattgraham</a></small></p>
-      </footer -->
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
+    <title>Tabfern</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.3.1/css/bootstrap.min.css">
+    <link rel="manifest" href="/manifest.json">
+    <link rel="stylesheet" href="/assets/css/styles.min.css">
+<link href="https://fonts.googleapis.com/css?family=Josefin+Sans&display=swap" rel="stylesheet"><link href="https://fonts.googleapis.com/css?family=Josefin+Sans&display=swap" rel="stylesheet">
+</head>
+
+<body style="overflow-x: hidden !important;">
+    <nav class="navbar navbar-light navbar-expand-md navbar-expand-xl sticky-top" style="background-color: white;">
+        <div class="container-fluid"><a class="navbar-brand" href="#"><img src="/assets/img/fern128.png" style="width: 50px;"></a><button data-toggle="collapse" class="navbar-toggler" data-target="#navcol-1"><span class="sr-only">Toggle navigation</span><span class="navbar-toggler-icon"></span></button>
+            <div
+                class="collapse navbar-collapse" id="navcol-1" style="justify-content: flex-end;">
+                <ul class="nav navbar-nav nav-list">
+                    <li class="nav-item" role="presentation"><a class="nav-link" href="#usage">Usage</a></li>
+                    <li class="nav-item" role="presentation"><a class="nav-link" href="#backup">Backup &amp; Restore</a></li>
+                    <li class="nav-item" role="presentation"><a class="nav-link" href="#limit">Limitations</a></li>
+                    <li class="nav-item" role="presentation"><a class="nav-link" href="#related">Privacy policy</a></li>
+                    <li class="nav-item" role="presentation"><a class="nav-link" href="https://github.com/cxw42/TabFern">View on Github</a></li>
+                    <li class="nav-item" role="presentation"><a class="nav-link" href="https://chrome.google.com/webstore/detail/tabfern-tab-manager-and-b/hbajjpcdbninabigakflkhiogmmjaakm">Add to Chrome</a></li>
+                </ul>
+        </div>
+        </div>
+    </nav>
+    <div style="min-width: 100vw;font-family: 'Josefin Sans', sans-serif;background-color: #4C8BF5;">
+        <div class="row">
+            <div class="col">
+                <div style="display: flex;flex-direction: column;padding: 20px 5px;">
+                    <header style="margin: auto;padding: 20px 5px;">
+                        <h1 style="font-size: 80px;text-align: center;color: white;font-weight: 800;margin: 60px 0px;">TabFern</h1>
```

Comment:
You had no way of knowing this :) , but in #137 @Yasujizr used Mina Bold for the logo font in TabFern.  See https://github.com/cxw42/TabFern/issues/137#issuecomment-439660499 .  Would you please use Mina Bold for this header for consistency?  I like your other font choices; I'd just like this one to be consistent.

Reply:
Thank you for changing the font!  Unfortunately, it shows up as serif on my PC (I don't have Mina installed locally).  Would you please add the [Google Fonts CSS](https://fonts.google.com/specimen/Mina?selection.family=Mina) or a [copy of the file](https://github.com/google/fonts/blob/master/ofl/mina/Mina-Bold.ttf) plus appropriate styles?

![image](https://user-images.githubusercontent.com/10515894/68073161-91ba3500-fd63-11e9-9e52-5d7a2e709142.png)